### PR TITLE
README.md + logo picture

### LINKED ISSUE #3: README.md
Editing the readme.md file and adding a description of the project.

### PR DESCRIPTION
**Pull request by Varditli**
Edited the README.md file and uploaded the logo picture
Fixes: #3

### CODE REVIEW

**Comment by shakedch** [comment]
Fixes: #3

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
+<div align="center">
+
+![](Joboard/static/img/readme_pics/JoBoardlogo.jpeg)
+
+</div>
+
+# JoBoard Project
+
+Project Groot is an open-source garden management system. In its core, Project Groot is a regular automatic plant growing system: it reads data from humidity and light sensors and decides when to give the plants more water, less water, more light or less light. What differentiates Project Groot from other garden management systems, is that it allows users to write their own growth plans for different plants. Users can then share their growth plans through the Groot network and users can also download and review plans from other users. Project Groot can also manage various plants with different sensors and growth plans at the same time.
```
Comment (bardielle) [inline README.md:9]: Update the description here.... 
your project name is not Groot....

**Review by bardielle — CHANGES_REQUESTED** [review]
Please add the relevant issue to that PR

**Comment by bardielle** [comment]
@Varditli ?

**Comment by Varditli** [comment]
Fixes: #3

**Comment by bardielle** [comment]
@Varditli Please look at my comment - fix the README file with the relevant project description

**Pull request closed without merging**